docs/platform-integration/snippets/shared_1/Storage.cs

```diff
@@ -17,9 +19,20 @@ public StoragePage()
 
         Content = new VerticalStackLayout
         {
+            new Button { Text = "ToUpperFile",
+                Command = new Command(ToUpperFile) },
```

Comment:
If this is in a snippet it should probably be changed to use an async command, or an async relay command.